packages/hash/frontend/src/components/layout/PageHeader/SearchBar/index.tsx

```diff
@@ -46,16 +50,35 @@ const toBlockUrl = (searchPage: PageSearchResult): string => {
   return segments.join("");
 };
 
-const ResultList: React.FC = (props) => (
-  <ul
-    className={tw`absolute z-10 w-1/2 max-h-60 overflow-auto border border-gray-100 rounded-lg shadow-md`}
+const ResultList: React.FC<{
+  isMobile: boolean;
+}> = ({ isMobile, ...props }) => (
+  <Box
+    component="ul"
+    sx={(theme) => ({
+      position: !isMobile ? "absolute" : "unset",
+      zIndex: 10,
+      width: "100%",
+      maxHeight: "15rem",
+      overflow: "auto",
+      border: `1px solid ${theme.palette.gray[20]}`,
+      borderRadius: "",
+    })}
+    className={tw`z-10 w-1/2 max-h-60 overflow-auto border border-gray-100 rounded-lg shadow-md`}
     {...props}
   />
```

Comment:
I appreciate this wasn't introduced as part of your PR, but can we use `props.children` directly instead of spreading props here? It would [resolve this](https://github.com/hashintel/hash/pull/358/files#r817657836)
```suggestion
  />
    {props.children}
```